{"name": "Логин с неверным логином и паролем", "status": "passed", "start": 1729340740158, "stop": 1729340740643, "uuid": "c166c9a7-ede0-4f2f-9630-d7d2be4c9628", "historyId": "fbeae77312463979bb58e896f211518f", "testCaseId": "fbeae77312463979bb58e896f211518f", "fullName": "tests.test_user_login.TestUserLogin#test_login_with_invalid_credentials", "labels": [{"name": "feature", "value": "Логин пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "Liana_M"}, {"name": "thread", "value": "14772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}